### ISSUE TITLE
Space in source Directory input path are not escaped 

### ISSUE THREAD

**Issue by andreacapriplexure** [piraces/azure-devops-bicep-task#1]
This is my steps
```
steps:
- task: BicepInstall@0
- task: BicepBuild@0
  inputs:
    sourceDirectory: "D://Arm Templates/deploy.bicep'"
```

**Comment by andreacapriplexure** [comment]
It then fails with

**Comment by piraces** [comment]
Thank you for reporting the problem. I will take a look to it and provide a solution as soon as possible

**Comment by piraces** [comment]
@andreacapriplexure this should be fixed in the version `0.1.2` that is already published in the marketplace. Make sure your pipeline use this last version. If you encounter further problems please reopen the issue.
